test/test.js

```diff
@@ -15,3 +17,27 @@ test('Basic test', (t) => {
     t.end()
   })
 })
+
+test('Cache directory is moved to new location', (t) => {
+  // Download to old cache directory location
```

Comment:
So ...? Apart from cleaning up the file it does the same as the steps you proposed right?